### ISSUE TITLE
Implement RTL support

### ISSUE THREAD

**Issue by srambach** [patternfly/react-topology#116]
**Describe the problem**
PatternFly now provides RTL language support, but topology has a few instances where it's causing problems. Node labels are one trouble spot.

**How do you reproduce the problem?**
Add `dir="rtl"` to the `<body>` tag to see it in RTL. Note that English text will still be readable left to right, but should be right aligned where it was previously left aligned, etc.

**Screenshots**
<img width="500" alt="image" src="https://github.com/patternfly/react-topology/assets/19825616/fe7d9f05-a345-4da6-b70e-9c623131e100">

**Any other information?**
See the RTL handbook for help - in particular, the helper multiplier might be of use.  https://www.patternfly.org/developer-resources/right-to-left-handbook#patternfly-rtl-helpers

**Comment by jeff-phillips-18** [comment]
@srambach Would RTL expect node label elements to be completely reversed?

LTR:   `<Icon> <Badge> <Label> <Context Menu>`

![image](https://github.com/patternfly/react-topology/assets/11633780/6ae92c82-3606-4bfd-b400-309654d49f06)

**Comment by jeff-phillips-18** [comment]
@andrew-ronaldson ^^

**Comment by srambach** [comment]
@jeff-phillips-18 Yes - I think those elements would be reversed in position as you list. Just FYI to clarify, unless you actually translate the text into an RTL language, though, the words themselves will read left to right. But yes, I'd expect the elements to reverse to be `<Context Menu> <Label> <Badge> <Icon>`.

**Comment by andrew-ronaldson** [comment]
What @srambach said! Thanks Sarah!
